packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/composables/useBranchLayout.ts

```diff
@@ -202,13 +202,9 @@ export function useBranchLayout({ phase, readonly, flowId }: { phase: NodePhase,
         return undefined
       }
 
-      const computedPosition = flowNode?.computedPosition
-      const absoluteX = computedPosition?.x ?? member.position.x
-      const absoluteY = computedPosition?.y ?? member.position.y
 
       rects.push({
-        x: absoluteX,
-        y: absoluteY,
+        ...member.position,
```

Comment:
`computedPosition` may not be latest value so we should use `member.position` directly.